CSS/CSS.sublime-syntax

```diff
@@ -271,6 +278,36 @@ contexts:
         - match: '\s*(?=[{;])'
           pop: true
         - include: media-query
+# Namespace At-Rule: https://www.w3.org/TR/css3-namespace/
+  namespace:
+    - match: \s*((@)namespace)\s*\b(\S*)
+      captures:
+        1: keyword.control.at-rule.namespace.css
+        2: punctuation.definition.keyword.css
+        3: entity.other.namespace-prefix.css
```

Comment:
The other at-rules had defined `@` as `punctuation`. Should the scopes also be removed from those? See lines 44, 121, 132, 144, 186, and 215.